support/mock/mockrt.go

```diff
@@ -297,6 +315,41 @@ func (rt *Runtime) Store() adt.Store {
 
 ///// Mocking facilities /////
 
+type message struct {
+	// message values
+	to     addr.Address
+	method abi.MethodNum
+	params abi.MethodParams
+	value  abi.TokenAmount
+
+	// returns from applying message
+	sendReturn runtime.SendReturn
+	exitCode   exitcode.ExitCode
+}
+
+func (m *message) Key() string {
+	return fmt.Sprintf("to: %v from: %v params: %v value: %v", m.to.String(), m.method, m.params, m.value)
+}
```

Comment:
I don't like the way this is abused as an informative message and a comparison key with a half-constructed other message. We can clean this up.

For verifying the expectation, instead write a `verify(t, to, from, etc...)` method